circleci: failed tests: TestDockerFinagle

The following test appears to have failed:

[#16096](https://circleci.com/gh/cockroachdb/cockroach/16096):

```
	at org.junit.runner.JUnitCore.run(JUnitCore.java:117)
	at org.junit.runner.JUnitCore.runMain(JUnitCore.java:98)
	at org.junit.runner.JUnitCore.runMainAndExit(JUnitCore.java:53)
	at org.junit.runner.JUnitCore.main(JUnitCore.java:45)

FAILURES!!!
Tests run: 12,  Failures: 1

I160408 17:41:45.154875 acceptance/cluster/localcluster.go:649  stopping
I160408 17:41:45.155124 acceptance/cluster/localcluster.go:547  event stream done, resetting...: net/http: request canceled
I160408 17:41:45.155137 acceptance/cluster/localcluster.go:570  events monitor exits
I160408 17:41:45.157146 acceptance/cluster/docker.go:397  ContainerKill: *errors.errorString: Error response from daemon: Cannot kill container 21713676ff523f4fa08132b70549a45c183b94f79a10454fb7e6afadac5d8e56: Container 21713676ff523f4fa08132b70549a45c183b94f79a10454fb7e6afadac5d8e56 is not running
--- FAIL: TestDockerFinagle (15.17s)
	util_test.go:281: expected success: non-zero exit code: 1
=== RUN   TestGossipPeerings
I160408 17:41:47.528753 acceptance/cluster/localcluster.go:286  Initializing Cluster AdHoc 3x1:
{"name":"AdHoc 3x1","nodes":[{"count":3,"stores":[{"count":1,"max_ranges":0}]}],"duration":5000000000}
I160408 17:41:47.530530 acceptance/cluster/docker.go:102  ImagePull cockroachdb/builder:20160305-182433 already exists
I160408 17:41:48.507949 acceptance/cluster/localcluster.go:581  creating certs (1024bit) in: /home/ubuntu/cockroach/.localcluster.certs.034701933
I160408 17:41:48.595662 acceptance/cluster/localcluster.go:532  events monitor starts
I160408 17:41:48.962486 acceptance/cluster/localcluster.go:480  *** started roach0 ***
  ui:        https://127.0.0.1:32830
  trace:     https://127.0.0.1:32830/debug/requests
--
--- SKIP: TestBuildBabyCluster (0.00s)
	terraform_test.go:30: only enabled during testing
=== RUN   TestFiveNodesAndWriters
--- SKIP: TestFiveNodesAndWriters (0.00s)
	util_test.go:65: running in docker mode
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

--- a/acceptance/finagle_test.go
+++ b/acceptance/finagle_test.go
@@ -17,7 +17,6 @@ package acceptance
 import "testing"
 
 func TestDockerFinagle(t *testing.T) {
-	t.Skip("TODO(dt): #5951, #5928, and needs #5484")
 	testDockerSuccess(t, "finagle", []string{"/bin/sh", "-c", finagle})
 }
 

